core/compiler/src/main/scala/com/fortysevendeg/exercises/compiler/codeGen.scala

```diff
@@ -0,0 +1,142 @@
+package com.fortysevendeg.exercises
+package compiler
+
+import scala.language.implicitConversions
+import scala.reflect.api.Universe
+
+case class CodeGen[U <: Universe](override val u: U = scala.reflect.runtime.universe) extends TreeHelpers[U] {
+  import u._
+
+  def emitExercise(
+    name: Option[String]
```

Comment:
I think this is XCode's fault (because it automatically does this) when I started doing some iOS work years ago. I really despised this kind of formatting, but ultimately I found it more readable and ended up doing it sometimes in Scala.

I'm 100% for reformatting it so that our codebase is uniform.